ui/panels-plugin/src/plugins/time-series-chart/VisualOptionsEditor.tsx

```diff
@@ -44,12 +44,12 @@ export function VisualOptionsEditor({ value, onChange }: VisualOptionsEditorProp
         control={
           <Slider
             data-testid={VISUAL_CONFIG.point_radius.testId}
-            defaultValue={DEFAULT_POINT_RADIUS}
+            value={value.point_radius ?? DEFAULT_POINT_RADIUS}
             valueLabelDisplay="auto"
             step={0.5}
             marks
             min={0}
-            max={20}
+            max={5}
```

Comment:
Optional suggestion: It could be nice to pull this into a constant in the model like we do for some other values that are quasi-synced up with what's in cue. Maybe something like `MAX_POINT_RADIUS` or an object like `POINT_RADIUS = { min: 0, max: 5}` or an array with two numbers like `POINT_RADIUS = [0, 5]`.

Reply:
Yeah good call, I just [added this in similar way](https://github.com/perses/perses/pull/798/commits/3e22f061eff55e029a8799521decc4fd0e779995) to your second example